sigle/src/modules/explore/Explore.tsx

```diff
@@ -11,7 +11,7 @@ export const ExploreUsers = () => {
   const { user, isLegacy } = useAuth();
   const { isLoading: isLoadingUserFollowing, data: userFollowing } =
     useGetGaiaUserFollowing({
-      enabled: !!user && !isLegacy,
+      enabled: !!user || !isLegacy,
```

Comment:
can you please explain this change?

Reply:
ah yes so this is relating to the infinite load issue I mentioned when a user is logged out - it appears `!!user && isLegacy` is causing the query to remain in a loading state so I assumed it's trying to fetch which is why used the `||` operator to force it to resolve with an error.

However, after reading the docs just now I realise that `isLoading` doesn't necessarily mean a fetch is happening, so I'm wondering if it would be better to revert the change and use `fetchStatus` [here](https://github.com/sigle/sigle/blob/5c727d9558ec7901bffdf3c8a4fb71b9d127842f/sigle/src/modules/explore/Explore.tsx#L71) instead of `isLoadingUserFollowing` ? (tested it and is working 👍)

Reply:
Sounds good 👍